src/utils/optionsdict.h

```diff
@@ -181,27 +202,49 @@ template <typename T>
 T OptionsDict::Get(const OptionId& option_id) const {
   return Get<T>(GetOptionId(option_id));
 }
-
 template <typename T>
-bool OptionsDict::Exists(const std::string& key) const {
+std::optional<T> OptionsDict::OwnGet(const std::string& key) const {
   const auto& dict = TypeDict<T>::dict_;
   auto iter = dict.find(key);
-  if (iter != dict.end()) return true;
-  if (!parent_) return false;
-  return parent_->Exists<T>(key);
+  if (iter != dict.end()) {
+    return iter->second.Get();
+  }
+  return {};
+}
+template <typename T>
+std::optional<T> OptionsDict::OwnGet(const OptionId& option_id) const {
+  return OwnGet<T>(GetOptionId(option_id));
+}
+
+template <typename T>
+bool OptionsDict::Exists(const std::string& key) const {
+  if (OwnExists<T>(key)) return true;
```

Comment:
aliases contains this - so this is redundant right?

Reply:
Indeed, removed.